src/components/view-center.js

```diff
@@ -0,0 +1,48 @@
+import { x } from '@xstyled/styled-components'
+import BancoComida from '../iamges/banco-de-comida.jpeg'
+import { useNavigate } from 'react-router-dom';
+
+export default function ViewCenter() {
+    const navigate = useNavigate();
+    return (
+            <x.div 
+                onClick={() => navigate(`./center/nombreCenter`)}
+                w={{ _:'19rem', lg:'30rem' , xl:'40rem' }} 
+                h={{_:'8rem', lg:'11rem', xl:'14rem' }}
+                mb={{_:'2rem', lg:'3rem'}}
+                position='relative'
+                display='flex'
+                alignItems='center'
+                justifyContent='space-evenly'
+                boxShadow='0px 4px 20px rgba(0, 0, 0, 0.1)'
+                borderRadius='20px'
+            >
+                <x.img
+                    src={BancoComida}
```

Comment:
It's better to import the image from the public folder as follows:

```suggestion
                    src="/image.type"
```

By default, the public folder path is `/` for all images.